### PR TITLE
added phone number verification method with twilio helper to business…

### PR DESCRIPTION
**Pull request by nooralnabi**
… controller

### CODE REVIEW

File: lib/twilio_helpers.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module TwilioHelpers
+  class << self
+    def send_verification_code(phone_number, code)
+      begin
+        client = Twilio::REST::Client.new
+        client.messages.create({
+          from: ENV['TWILIO_NUMBER'],
+          to: phone_number,
+          body: "This is your Pure Platform application verification code: #{code}",
```
Comment (MwMaki) [inline lib/twilio_helpers.rb]: Do we need translation for this message?

Comment (nooralnabi) [reply lib/twilio_helpers.rb]: @ssharba we need the translation now or in next steps return to all the API and added  translation

Comment (ssharba) [reply lib/twilio_helpers.rb]: let's add translation please

File: config/application.rb [hunk 2]
```diff
@@ -21,15 +21,8 @@ module PureBusinessApi
...
-    # Application configuration can go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded after loading
-    # the framework and any gems in your application.
-
-    # Only loads a smaller set of middleware suitable for API only apps.
-    # Middleware like session, flash, cookies can be added back manually.
-    # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+    config.autoload_paths << Rails.root.join('lib')
+    config.i18n.default_locale = :en
```
Comment (MwMaki) [inline config/application.rb]: make default_locale =  ar  and use Baghdad time as default please

File: app/controllers/api/v1/business_infos_controller.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class Api::V1::BusinessInfosController < ApplicationController
+
+  def create
+    business= Business.find(params[:business_id])
+    raise "#{I18n.translate('activerecord.attributes.controller.messages.business_has_profile')}" if business.business_info
```
Comment (MwMaki) [inline app/controllers/api/v1/business_infos_controller.rb]: I think this should be
raise "......."    unless  business.business_info.present?

File: app/controllers/api/v1/businesses_controller.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
+class Api::V1::BusinessesController < ApplicationController
+  skip_before_action :authenticate_business_with_token!, only: [:phone_number_verification]
+
+  def phone_number_verification
+    if params[:phone_number].present? && params[:verification_code].present? 
+      business_verification_phone=PhoneVerification.find_by_phone_number_and_verification_code(params[:phone_number],params[:verification_code] )
+      if business_verification_phone.present?
+        business_verification_phone.update(is_verified: true)
+        render json: {status: true , message: 'ok',phone_number: business_verification_phone.phone_number,  }, status: :ok
```
Comment (MwMaki) [inline app/controllers/api/v1/businesses_controller.rb]: translate the "message"

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -2,10 +2,30 @@ class ApplicationController < ActionController::API
...
   end
+  
+  def set_locale
+    if params[:locale]
+      I18n.available_locales.map(&:to_s).include?(params[:locale]) ? I18n.locale = params[:locale] : I18n.locale = I18n.default_locale
+    else
+      params[:locale] = I18n.default_locale
+      I18n.locale = I18n.default_locale
+    end
+  end
```
Comment (MwMaki) [inline app/controllers/application_controller.rb]: I don't know if it's the right task or not to mention this in, but don't forget the globalize fallback for local

File: app/controllers/api/v1/business_infos_controller.rb (hunk 3 above)
Comment (nooralnabi) [reply app/controllers/api/v1/business_infos_controller.rb]: @MwMaki this error to check if we have business_info will raise an error

File: app/controllers/application_controller.rb (hunk 5 above)
Comment (nooralnabi) [reply app/controllers/application_controller.rb]: for this task, no need to put the globalize fallback

Comment (MwMaki) [reply app/controllers/application_controller.rb]: Ok, but Please keep it in mind so not to be forgotten.

File: app/controllers/api/v1/businesses_controller.rb (hunk 4 above)
Comment (nooralnabi) [reply app/controllers/api/v1/businesses_controller.rb]: @MwMaki what we need to put her in this message after the business verified his number

Comment (MwMaki) [reply app/controllers/api/v1/businesses_controller.rb]: Anything you find suitable like,
"Thank you for verifying your account."

**Pull request merged**